main.py

```diff
@@ -4,12 +4,26 @@
 import os
 from helper import *
 from discord_helpers import leaderboard, hasRole
+import json
 
 DATABASE_NAME = 'votes.db'
-REACTION_TO_COMPARE = '🐻'
+REACTION_TO_COMPARE = ['🐻'] if not 'DISCORD_ALLOWED_REACTION' in os.environ else json.loads(os.environ['DISCORD_ALLOWED_REACTION'])
 LEADERBOARD_LIMIT = 10
 BOT_TOKEN = os.environ['DISCORD_BOT_TOKEN']
 REQUIRED_ROLE_ID = os.environ['ROLE_ID']
+NOTIFY_USER = os.environ['NOTIFY_USER']
+IGNORED_CHANNELS = [763798356484161569, 772838189920026635,  839229026194423898] if not 'DISCORD_IGNORED_CHANNELS' in os.environ else json.load(os.environ['DISCORD_IGNORED_CHANNELS'])
+# defaults to General, Lumenauts, Report-spam
+
+
+if not (isinstance(REACTION_TO_COMPARE, list) or  len(REACTION_TO_COMPARE) != 0 and not isinstance(REACTION_TO_COMPARE[0], str)):
+    # REACTION_TO_COMPARE must be str to arr of str; empty array => wildcard 
+    print("DISCORD_ALLOWED_REACTION env variable has to be array of strs!")
+    exit
+if not isinstance(IGNORED_CHANNELS, list) or (len(IGNORED_CHANNELS) != 0 and not isinstance(IGNORED_CHANNELS[0], int)):
```

Comment:
We leave spaces between the ifs :)

Reply:
What exactly do you mean?
Splitting the logic up into nested if-s or new lines where they are logical?